includes/class.llms.ajax.handler.php

```diff
@@ -712,6 +712,14 @@ public static function select2_query_posts() {
 		}
```

Comment:
changelogs to file header and method

Reply:
fixed in https://github.com/gocodebox/lifterlms/pull/832/commits/bc7ca7152db7783d7ca1b91c4471ed8389acb90a